### ISSUE TITLE
Generate fake data for dev/staging databases

### ISSUE THREAD

**Issue by alanmoo** [mozilla/network-pulse-api#196]
It's frustrating to have to spin up a database and populate it with data after previously nuking a dev database. There should be an easier way to test PRs locally and in review apps. Can we have build scripts that generate this content for us?

@gideonthomas

**Comment by Pomax** [comment]
This would effectively be something like 
```
> pg_dump prod-db-ui > data.pg
> <update data.pg so that its heroku-random-superuser-name becomes the local dev psql superuser>
> dropdb network-pusle-api
> createdb network-pusle-api
> psql network-pulse-api
psql> \i data.pg
psql> \q
```

**Comment by cadecairos** [comment]
This could be done with an approach similar to what we did in mozilla/network#783

**Comment by alanmoo** [comment]
How fragile is the approach used in that ticket, @cadecairos? I'm wondering if using something like [Faker](http://faker.readthedocs.io/en/master/) might be more maintainable, as we can test with it and know if updates to our model require updates to our mock data.

**Comment by xmatthewx** [comment]
Thanks for poking at this. I wonder if there's a way to use real data, since it can help catch quirks. Maybe grabbing a dump of the 50 latest entries and their associated people, tags, etc.

**Comment by alanmoo** [comment]
That's kind of what we did with the Network/Foundation site, but I was thinking generating fake data might be more useful for catching quirks in layout based around edge cases we might not have in a database already. But I don't give too many 🥄 about how we do it, just that we have something that makes it easier to dev/test locally.

**Comment by cadecairos** [comment]
> How fragile is the approach used in that ticket, @cadecairos? I'm wondering if using something like Faker might be more maintainable

Yes, what we've got now will require us to do a dump of data + cleanup every time we do a schema change.

With something like Faker, we'll still need to adjust our scripts after a schema change, but it should be less painful.

**Comment by alanmoo** [comment]
Yup, that's the other thing I was thinking- with Faker the data becomes more a part of our testing process than the problems with dumping and migrating, which has the potential to...go stale if we don't keep on top of it.

**Comment by cadecairos** [comment]
This should be modelled after the work done to add FactoryBoy+Faker on the foundation site: mozilla/foundation.mozilla.org#932
